front/app/containers/Admin/dashboard/users/Charts/AreaChart.tsx

```diff
@@ -21,53 +21,57 @@ interface Props {
   endAt: string | null;
   currentGroupFilter: string | undefined;
   currentGroupFilterLabel: string | undefined;
+  customFieldId: string;
 }
 
-export const fallbackMessages = {
+const fallbackMessages = {
   _blank: messages._blank,
-  outside: messages.otherArea,
 };
 
-const AreaChart = (
-  props: Props & WrappedComponentProps & InjectedLocalized
-) => {
-  const {
-    intl: { formatMessage },
-    localize,
-  } = props;
-
-  const { data: usersByDomicile } = useUsersByDomicile({
-    start_at: props.startAt,
-    end_at: props.endAt,
-    group: props.currentGroupFilter,
-    enabled: true,
+const AreaChart = ({
+  intl: { formatMessage },
+  localize,
+  startAt,
+  endAt,
+  currentGroupFilter,
+  currentGroupFilterLabel,
+  customFieldId,
```

Comment:
Same comment as for the other PR. I wonder if it should not be the responsibility of the component to get the custom field id.